coopr-docs/docs/source/rest/node-usage.rst

```diff
@@ -0,0 +1,174 @@
+..
+   Copyright © 2012-2015 Cask Data, Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+ 
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+:orphan:
+
+.. index::
+   single: Metrics 
+
+=======
+Metrics
```

Comment:
Is this file really about Metrics, or would a better title be "Node Usage Metrics"?